ui/k-info-panel/edit_window.kytos

```diff
@@ -127,57 +127,39 @@
             saveWindow: function() {
                 var _this = this
 
-                // Translate the status in the window to a 0,1,2 number used
-                // in the maintenance API.
-                var status
-                if(_this.window_data.status == 'Pending') {
-                    status = 0
-                } else if(_this.window_data.status == 'Running') {
-                    status = 1
-                } else if(_this.window_data.status == 'Finished') {
-                    status = 2
-                } else {
-                    status = _this.window_data.status
-                }
-
-                jsonItems = []
+                filteredLinks = []
+                filteredSwitches = []
+                filteredInterfaces = []
+                
                 // For every chosen link
-                for(i = 0; i < this.chosen_links.length; i++) {
+                for(item of this.chosen_links) {
```

Comment:
@Ktmi, if you were to try to edit and save the name of the MW, when the expected tz format is resulting an error, could you look into it?

![20221222_135703](https://user-images.githubusercontent.com/1010796/209187095-5bd7fef4-39d0-4108-9138-d65921659047.png)

Reply:
I'm not sure if the date format issue was fixed, but I'm facing another issue when editting the table:

![Screenshot 2023-01-13 at 10 29 26 AM](https://user-images.githubusercontent.com/168159/212331354-79fab045-ed64-4247-ae46-c62d8668645f.png)

I'm using the most updated version of the UI (including the changes from PR https://github.com/kytos-ng/ui/pull/33)

Reply:
This issue was fixed in the last commits. The issue I was having is related to https://github.com/kytos-ng/ui/issues/34